docs/kratos/manage-identities/05_identity-schema.mdx

```diff
@@ -1,11 +1,11 @@
 ---
 id: identity-schema
 title: Identity Schema - the model that defines the data that makes up user accounts
-sidebar_label: Identity Schema
+sidebar_label: Identity schema
 toc_max_heading_level: 4
 ---
 
-# Identity Schema
```

Comment:
This hasn't made its way into the guidelines, but after some talks with @vinckr we wanted to make "Identity Schema" capitalized since it's an Ory-specific term. I'm not 100% set on that so:

a) you can revert your changes and keep it capitalized the way it was
b) make sure that (at least in this doc) you remove capitalization from all Identity Schema references (there are more than those two you edited)

PS. Titles and headings must be sentence case, but if they use terms that are proper names or are traditionally capitalized then it's okay to have them capitalized, too.

Reply:
Let's use regular captialization IMO. I think ory specific terms would most likely be prefixed with ory like Ory Session token

Reply:
done